forc-plugins/forc-doc/src/render.rs

```diff
@@ -51,11 +51,12 @@ impl RenderedDocumentation {
             ),
             None => panic!("Project does not contain a root module"),
         };
+        println!("{root_module:#?}");
```

Comment:
```suggestion
```

Or we can use debug log here if desirable

Reply:
I just forgot to take this one out with the others